events.html

```diff
@@ -40,7 +40,7 @@ <h1>Events</h1>
             {% endif %}
           </p>
           <h2>{{ featured.title }}</h2>
-          <a href="{{ featured.url | relative_url }}">Read More</a>
```

Comment:
This seems odd, we are changing the text to *Register now* but we had *Read More* 

I think the text should stay *Read More*, right?

Reply:
@therapon I just pushed a fix for this. All the events CTA are using the "Register now" label. I forgot to change this